src/pages/index.js

```diff
@@ -1,4 +1,3 @@
-import * as constants from "../utils/constants.js";

```

Comment:
Needs correction: the width of the avatar src input should be equal to the form inner width